service/gcp_streamer_internal_test.go

```diff
@@ -12,14 +12,16 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-package tracker
+package service
 
 import (
-	"github.com/onflow/flow-go/model/flow"
+	"testing"
 )
 
-type Payload struct {
-	Header     *flow.Header
-	Guarantees []*flow.CollectionGuarantee
-	Seals      []*flow.Seal
+func TestNewGCPStreamer(t *testing.T) {
```

Comment:
Are these functions still needed?

Reply:
What functions are you referring to? The `TestNewX` ones?

Reply:
Yes, these two functions are here but empty, and I can see that there are identical ones in `service/cloud/gcp_streamer_internal_test.go`, so I guess this might have been a file that was moved..?

Reply:
Indeed, it's a mistake and shouldn't be there :p